ActindoWorkFlow/Integration/01_ConfigureQueueTypes.md

```diff
@@ -0,0 +1,373 @@
+- [Manage the workflows](/ActindoWorkFlow/Operation/01_ManageWorkflows.md)
+
+
+
+
+## Assign a queue type
+
+Assign a certain queue type to a transition, for instance to prioritize a certain action to be executed faster.
+
+### Prerequisites
+
+At least one queue type is created, see [Create a queue type](#create-a-queue-type).
+
+### Procedure
+
+*Workflows > Workflows > Tab OVERVIEW > Select workflow > Select workflow version*
+
+![Workflow editor](/Assets/Screenshots/ActindoWorkFlow/Workflows/WorkflowEditor.png "[Workflow editor]")
+
+1. Select the transition you want to assign a certain queue type to.      
+The transition is highlighted and its settings are displayed in the settings side bar on the right.
+
+  ![Action](/Assets/Screenshots/ActindoWorkFlow/Workflows/Action.png "[Action]")
+
+2. Click the *Queue Type* drop-down list and select the desired queue type for the selected transition. All available queue types are displayed in the list.     
+The selected queue type is displayed in the list.
+
+  > [Info] Repeat the steps **1** and **2** for all transitions to which you want to assign a queue type.
+
+3. Click the ![Points](/Assets/Icons/Points02.png "[Points]") (Points) button in the upper left corner next to the workflow name.   
+  The workflow context menu is displayed.
+
+  ![Context menu](/Assets/Screenshots/ActindoWorkFlow/Workflows/ContextMenu.png "[Context menu]")
+
+4. Click the *Deploy* menu entry in the context menu.   
+  A new workflow version with the new assigned queue types is deployed and published.
+
+### Next steps
+
+- [Delete a queue type](#delete-a-queue-type)
+- [Kill workers](#kill-workers)
+- [Configure the logging setting](01_ConfigureLoggingSettings.md)
+
+### See also
+
+- [User Interface workflows](/ActindoWorkFlow/UserInterface/00_UserInterface.md)
+- [Create a queue type](#create-a-queue-type)
+- [Edit a queue type](#edit-a-queue-type)
+- [Workflow and process elements](/ActindoWorkFlow/Overview/02_WorkflowProcessElements.md)
+- [Manage the workflows](/ActindoWorkFlow/Operation/01_ManageWorkflows.md)
+
+
+
+## Delete a queue type
+
+You can delete a queue type that is obsolete to release workers for other queue types. Be aware that the *Default* queue type cannot be deleted.
+
+### Prerequisites
+
+- At least one queue type is created, see [Create a queue type](#create-a-queue-type).
+- The workflows are disabled, see [Disable the workflows](#disable-the-workflows).
+- No more workers are running.
+
+### Procedure
+
+*Workflows > Queue Types > Tab OVERVIEW*
+
+![Overview](/Assets/Screenshots/ActindoWorkFlow/QueueTypes/Overview.png "[Overview]")
+
+1. Select the checkbox of the queue type you want to delete in the list of queue types.      
+The editing toolbar is displayed above the queue types list.
+
+  ![Toolbar](/Assets/Screenshots/ActindoWorkFlow/QueueTypes/Toolbar.png "[Toolbar]")
+
+2. Click the ![Delete](/Assets/Icons/Trash03.png "[Delete]") (Delete) button in the toolbar.   
```

Comment:
``Click the ![Delete](/Assets/Icons/Trash03.png "[Delete]") (Delete) button in the toolbar.  

In UI steht DELETE (Stand 29.06.).